test/policies/organization_policy_test.exs

```diff
@@ -0,0 +1,66 @@
+defmodule CodeCorps.OrganizationPolicyTest do
+  use CodeCorps.PolicyCase
+
+  import CodeCorps.OrganizationPolicy, only: [create?: 1, update?: 2]
+
+  describe "create" do
+    test "returns true when user is an admin" do
+      user = build(:user, admin: true)
+
+      assert create?(user) == true
+    end
+
+    test "returns false when user is not an admin" do
+      user = build(:user, admin: false)
+
+      assert create?(user) == false
+    end
+  end
+
+  describe "update" do
+    test "returns true when user is an admin" do
+      user = insert(:user, admin: true)
+      organization = insert(:organization)
+
+      assert update?(user, organization) == true
+    end
+
+    test "returns false when user is not member of organization" do
+      user = insert(:user)
+      organization = insert(:organization)
+      assert update?(user, organization) == false
+    end
+
+    test "returns false when user is pending member of organization" do
+      user = insert(:user)
+      organization = insert(:organization)
+      insert(:organization_membership, role: "pending", member: user, organization: organization)
```

Comment:
We seem to have this block repeated in a few places where only the role varies. Thoughts on extracting this to a private method?

```
defp setup_user_organization_by_role(role) do
  ... all the codes but we pass in "pending"
end
```
This saves us a few lines but adds readability to the test so we can know exactly the intention of this block.